### PR TITLE
Enable reco for ILD_FCCee_v01/2

### PR DESCRIPTION
**Pull request by Victor-Schwan**
BEGINRELEASENOTES
- minimal requirements for reco with ILD_FCCee_v01/2 models satisfied
ENDRELEASENOTES

### CODE REVIEW

**Review by tmadlener — APPROVED** [review]

**Comment by tmadlener** [comment]
Do you want to add a run for this on CI? Or is it not yet ready?

File: .github/workflows/key4hep.yml [hunk 1]
```diff
@@ -87,13 +87,13 @@ jobs:
                   --outputFileBase=bbudsc_3evt_GaudiRec \
                   --detectorModel=${{ matrix.detector_model }}
 
-  # Reconstruction with the v11 model (FCCee) using EDM4hep inputs and outputs
-  gaudi-v11-reco:
+  # Reconstruction with the ILD@FCCee models using EDM4hep inputs and outputs
+  gaudi-ILDatFCCee-reco:
```
Comment (tmadlener) [inline .github/workflows/key4hep.yml:91]:
```suggestion
  gaudi-ILDetFCCee-reco:
```
(?)

**Review by tmadlener — COMMENTED** [review]
We will probably have to think about a better solution for grouping the detector models than having different tuples and simply checking if the detector model is in there. However, I would do that in a separate PR.

File: .github/workflows/key4hep.yml (hunk 1 above)
Comment (Victor-Schwan) [reply .github/workflows/key4hep.yml:91]: I don't understand that. 'et' as in French for 'and'?

Comment (andresailer) [reply .github/workflows/key4hep.yml:91]: Probably ILDet(ector)

Comment (tmadlener) [reply .github/workflows/key4hep.yml:91]: Yes, currently it's `Dat` which I interpreted as a type for `Det`.

Comment (andresailer) [reply .github/workflows/key4hep.yml:91]: I think one is supposed to read ILD at FCCee...

Comment (tmadlener) [reply .github/workflows/key4hep.yml:91]: Aaaahhh, well that also makes sense. In that case I just misread that.

**Pull request merged**